### PR TITLE
Add provision for running all e2e tests

### PR DESCRIPTION
**Pull request by ashetty1**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -75,9 +75,9 @@ prepare-release: cross
 test:
 	go test -race $(PKGS)
 
-# Run e2e tests
-.PHONY: test-e2e
-test-e2e:
+# Run main e2e tests
```
Comment (kadel) [inline Makefile]: it would be good to describe what it "main e2e tests" mean

**Comment by cdrage** [comment]
LGTM :+1:

**Comment by cdrage** [comment]
We might have to merge this in and then modify this after in case @ashetty1 is away :100: 

But thanks for this fix!

**Comment by anmolbabu** [comment]
@kadel can we pull this into master?

**Comment by cdrage** [comment]
Let's merge :+1:

**Pull request merged**